CodeSniffer/Standards/PEAR/Sniffs/Commenting/FileCommentSniff.php

```diff
@@ -347,6 +347,12 @@ protected function processPackage(PHP_CodeSniffer_File $phpcsFile, array $tags)
             $newContent = preg_replace('/[^A-Za-z_]/', '', $newContent);
             $nameBits   = explode('_', $newContent);
             $firstBit   = array_shift($nameBits);
+            
+            if(!isset($firstBit{0})){
```

Comment:
Characters within strings may be accessed and modified by specifying the zero-based offset of the desired character after the string using square array brackets, as in `$str[42]`. Think of a string as an array of characters for this purpose.
Strings may also be accessed using braces, as in `$str{42}`, for the same purpose. 

More on [php.net](http://php.net/manual/en/language.types.string.php#language.types.string.substr).